### PR TITLE
[New] Add videojs-hotkeys@0.2.20 w/ git auto-update

### LINKED ISSUE #9818: [Request] Add videojs-hotkeys
**Library name:** videojs-hotkeys
**Git repository url:** https://github.com/ctd1500/videojs-hotkeys
**License(s):** Apache-2.0

For beginners who is interested in picking up this issue:

In this issue, we are going to host a new web front-end library on CDNJS, so that web developer can easily use it on our free CDN without additional download/upload process 🚀 .

If you'll like to work locally, I'll suggest you start pulling cdnjs repository with sparseCheckout: https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md, since there are so many files in this repo, without sparseCheckout, the whole process will be slow and will use more than 80GB disk space, using git command line can learn more skills and experience, if you don't like that, instead, you can also just do it on GitHub, it's more convenient, but it may be a little bit harder to fix problems on GitHub gui.

Here are the docs about how to add a library:
 - https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#d-adding-a-new-library-with-its-assets
 - https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson

We'll also ask to setup auto-update config so that the added libraries can be updated to the latest version easily:
 - https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

For beginners who is interested in join and help add this library, please feel free to tell me what kind of help you'll need, we'll help you send the valid pull request, thanks 😄

### PR DESCRIPTION
**Pull request by sashberd**
close #9818, cc @ctd1500

Pull request for issue: #9818 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ctd1500/videojs-hotkeys/tree/master
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/videojs-hotkeys
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 79
   - Count of forks: 59
 * NPM download stats (optional):
   - Downloads in the last day: 7
   - Downloads in the last week: 707
   - Downloads in the last month: 3287

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@sashberd Sorry for removing the `assets` part from your commit msg. The first line of commit msg had better be not longer the 50 characters.

File: ajax/libs/videojs-hotkeys/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "Chris Dougherty",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ctd1500/videojs-hotkeys.git"
+  },
+  "keywords": [
+    "videojs",
+    "videojs-plugin",
+    "hotkeys"
```
Comment (sufuf3) [inline ajax/libs/videojs-hotkeys/package.json]: @sashberd Maybe we can ad more keywords, such as `video`.
What do you think?
I refer it from https://github.com/ctd1500/videojs-hotkeys/blob/master/bower.json#L18
Thank you.

**Comment by sashberd** [comment]
@sufuf3 The PR was updated

**Comment by PeterDaveHello** [comment]
The commit message is obviously wrong since you're using git auto-update here but mention `npm` in https://github.com/cdnjs/cdnjs/pull/12031/commits/9ada6f564a2f59a5fd27df4c863be2281ebbb84a.

**Pull request merged**